core/src/test/resources/xform_tests/test_looped_model_iteration.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
+    <h:head>
```

Comment:
Would be nice to run the AS code formatter on this file. It makes it a bit more readable